src/hedgehog.js

```diff
@@ -127,6 +127,36 @@ class Hedgehog {
     }
   }
 
+  /**
+   * Confirms the user credentials given generate the same entropy after using artifacts from the server
+   * @param {String} username username
+   * @param {String} password user password
+   * @returns {boolean} whether or not the credentials are valid for the current user
+   */
+  async confirmCredentials (username, password) {
+    let self = this
+
+    const existingEntropy = WalletManager.getEntropyFromLocalStorage()
+    if (!existingEntropy) return false // not logged in yet
+
+    let lookupKey = await WalletManager.createAuthLookupKey(username, password)
+    let data = await self.getFn({ lookupKey })
```

Comment:
i know you're following the existing thing, but these should def be const

Reply:
Lol true, copy/paste spotted